src/dawn/IIR/Stencil.cpp

```diff
@@ -139,8 +156,8 @@ std::vector<Stencil::FieldInfo> Stencil::getFields(bool withTemporaries) const {
 
 std::vector<std::string> Stencil::getGlobalVariables() const {
   std::set<int> globalVariableAccessIDs;
-  for(const auto& multistage : multistages_) {
-    for(const auto& stage : multistage->getStages()) {
+  for(const auto& multistage : children_) {
+    for(const auto& stage : multistage->getChildren()) {
```

Comment:
1 iterator

Reply:
in later PR we are using iterateIIROver

Reply:
yeah, I just wanted to tag occurrences so we see them when merging backward